### ISSUE TITLE
Investigate method of generating reset alerts

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#155]
As part of #154, we need a generalized way to generate alerts once the 'reset all' button has been pressed.  As part of this, we may need to restrict alerts that come from other parts of the simulation once properties are reset.  ```aria-live``` elements might have text content reset, but we don't always want them to be announced.  ```assertive``` alerts will always take precedent, which is not desireable either.  So we need to generate a single alert when reset is pressed.

**Comment by jessegreenberg** [comment]
In accessibility meeting 9/20/16, we decided that it is acceptable to have a general alert when Reset All button is pressed.  The alert is 
> “Sim screen restarted, everything reset”

THis is good for i18n, is very clear that *everything* is reset, and it uses student friendly language.

In order to prioritize this alert over others that might happen when model properties are reset, perhaps we can do something like:

- Hide aria-live alerts so screen readers do not announce changes
- Full model reset
- Unhide aria-live elements
- Set aria-live text content to “Sim screen restarted, everything reset”

Assuming that aria-live elements are not announced when hidden, this should work on all screen readers and is generalizable when the feature is moved to scenery-phet ResetAllButton.

**Comment by jessegreenberg** [comment]
Done in above commit, needs screen reader testing.

**Comment by jessegreenberg** [comment]
Tested with on Windows with NVDA + Firefox and IE. Seems to work nicely, closing.
